src/accommodation/accommodation.service.ts

```diff
@@ -809,4 +843,16 @@ export class AccommodationService {
 
     return orderingObjsArray;
   }
+
+  private isDefaultOptions(options: OrderAndFilterDto): boolean {
+    return (
```

Comment:
@Zach13131 this part of code looks strange, maybe try this:
```
const {
    limit = CachingPagesLimit,
    minPrice = 0,
    maxPrice = AccommodationMaxPrice,
    minRooms = 0,
    maxRooms = AccommodationMaxRooms,
    minPeople = 0,
    maxPeople = AccommodationMaxPeople,
  } = options;

  return (
    limit === CachingPagesLimit &&
    minPrice === 0 &&
    maxPrice === AccommodationMaxPrice &&
    minRooms === 0 &&
    maxRooms === AccommodationMaxRooms &&
    minPeople === 0 &&
    maxPeople === AccommodationMaxPeople
  );
```

Reply:
yea, this looks much better thanks)